src/index.js

```diff
@@ -5,7 +5,6 @@ import { CompositeDisposable } from 'atom'
 import { get } from 'request-promise'
 
 const DEFAULT_ARGS = [
-  '--cache', 'false',
```

Comment:
@maschwenk it looks like you added this line in #187, do you remember the reasoning behind disabling the cache?

@vzamanillo I'm not sure this removal should be silently included in this PR without an explanation.

Reply:
@vzamanillo if this is incompatible with `--parallel` maybe adding it should be moved to an `else` block on that condition? (If it's something we want to keep in here in the first place.)

Reply:
@Arcanemagus I didn't actually add that, it just carried over from the conversion from coffee -> es6 https://github.com/AtomLinter/linter-rubocop/pull/187/files#diff-2290943f47872064e53f430a293f3c8fL11

Off the top of my head I can't imagine why you'd want that option.

Reply:
The question is why we would not cache and reuse results? looks like disabling cache does not bring any benefit. The docs says:

> Cache Validity
> 
> Later runs will be able to retrieve this information and present the stored information instead of inspecting the file again. This will be done if the cache for the file is still valid, which it is if there are no changes in:
> 
>     the contents of the inspected file
>     RuboCop configuration for the file
>     the options given to rubocop, with some exceptions that have no bearing on which offenses are reported
>     the Ruby version used to invoke rubocop
>     version of the rubocop program (or to be precise, anything in the source code of the invoked rubocop program)

so, why not caching?

Reply:
Whoops! Missed that in the hidden deleted file 🤦‍♂️.

Looks like it was initially added in https://github.com/AtomLinter/linter-rubocop/pull/72/commits/1c4282e0b79644f941d024b787689225ddb73f27, but there is no reasoning attached to it there. If it works with caching enabled and neither of you can think of a reason to keep it disabled I'm fine with it!

Reply:
Hmmm, had a thought based on the PR where that commit was introduced: Does the cache force `rubocop` to only lint the contents on disk, instead of respecting the contents we feed it?

Reply:
Looks like always lint over `stdin` contents.

Reply:
Maybe let's step back for a sec. This linter only runs on a single file through stdin in the first place. The parallelism is _most likely_ parallelized at the file level...so I'm not even sure this addition will do anything...in fact it may slow it down if it tries to spin up a thread to run on a single file.

Reply:
yeah.....this runs in parallel on _each file_ , so I think adding this option will actually do nothing, in fact I think it will be harmful https://github.com/rubocop-hq/rubocop/pull/4272/files#diff-68891125561f1767ea375a89b05f7e1eR53/

Reply:
Plus, parallel can not run with autocorrect.